tests/diff.yml

```diff
@@ -5,8 +5,8 @@ args: ["-timeout=30s", "-shuffle=on", "-race", "./..."]
 results:
   ferretdb:
     stats:
-      expected_fail: 53
-      expected_pass: 22
+      expected_fail: 67
```

Comment:
Dumb question, we added 6 tests so I understand `expected_pass` increased from `22` to `28`. Why does `expected_fail` increase from 53 to 67? :thinking: